tests/core/test_views_errors.py

```diff
@@ -197,6 +197,23 @@ def test_500_passes_exception_as_exc_info(self):
         self.assertEqual(mocked_logger.error.call_args[1]['exc_info'], custom_error)
 
 
+class ErrorPostgresViewTest(FormattedErrorMixin, PostgreSQLTest, unittest.TestCase):
+    sample_url = "/mushrooms"
+
+    def test_400_with_invalid_query_field(self):
```

Comment:
instead of having one test method that tests two things, it's better to have two methods:

- `test_400_when_query_field_contains_nul_character`
- `test_400_when_query_value_contains_nul_character`